cdap-explore/src/main/java/co/cask/cdap/explore/service/ExploreServiceUtils.java

```diff
@@ -154,6 +166,13 @@ public static HiveSupport checkHiveSupport(@Nullable ClassLoader hiveClassLoader
       }
     }
 
+    boolean useLatestVersionForUnknown = cConf.get(Constants.Explore.HIVE_AUTO_LATEST_VERSION).equals(
```

Comment:
No need to get `Constants.Explore.HIVE_AUTO_LATEST_VERSION` from `cConf`, it is a string literal.